datetime_test.go

```diff
@@ -82,6 +84,17 @@ func TestDayOfWeek(t *testing.T) {
 	}
 }
 
+func TestDayOfWeekReturnsDifferentValues(t *testing.T) {
+	dayMap := make(map[string]struct{})
+	iterations := 5 // sufficiently large to assure we don't randomly get the same value again
+	for i := 0; i < iterations; i++ {
+		day := GetDateTimer().DayOfWeek()
+		//t.Log(day)
+		dayMap[day] = struct{}{}
+	}
+	assert.True(t, len(dayMap) > 1)
```

Comment:
Referring to my first comment about testify, could you please change this into manual assertion without using testify? :D 

Thank you